src/test/groovy/NotificationManagerStepTests.groovy

```diff
@@ -1109,4 +1109,21 @@ class NotificationManagerStepTests extends ApmBasePipelineTest {
     printCallStack()
     assertFalse(assertMethodCallContainsPattern('githubCreateIssue', 'assignee'))
   }
+
+  @Test
+  void test_notify_pr_issue_1744() throws Exception {
+    script.notifyPR(
+      build: textOrFiletoJSON(file: "1744/build-info.json"),
+      buildStatus: "FAILURE",
+      changeSet: [],
+      statsUrl: "https://ecs.example.com/app/kibana",
+      stepsErrors: textOrFiletoJSON(file: "1744/steps-errors.json"),
+      testsErrors: [:],
+      testsSummary: []
+    )
+    printCallStack()
+    assertJobStatusSuccess()
+    assertFalse(assertMethodCallContainsPattern('githubPrComment', 'message=}'))
```

Comment:
Yes, I wanted to verify the message was empty, so the existing Unit Testing for Jenkins pipelines is a bit rudimentary in terms of the assertions, since it uses plain strings